### ISSUE TITLE
Adaptive Dark/AMOLED mode

### ISSUE THREAD

**Issue by ahmed-h-ayyad** [AllanWang/Frost-for-Facebook#1597]
Add adaptive mode to activate dark or AMOLED theme automatically at night or in low battery level.

OS Version: 4.9.112-perf+ (1574138402)
OS SDK: 28
Device (Manufacturer): RMX1921 (realme)
Model (Product): RMX1921 (RMX1921)
Package Installer: org.fdroid.fdroid
Tablet: false
Screen Dimensions: 1080 x 2264

App: com.pitchedapps.frost
App Version Name: 2.4.1
App Version Code: 2040100

Prev version: -1
Random Frost ID: 1576949396695-996400830-FO
Locale: English (United States)

**Comment by AllanWang** [comment]
This is possible, and something I actually looked at many years ago. I decided against providing my own UI for time selection + checking for battery levels, so it will likely be attached to Android Q's dark mode theme toggle. In short, supporting theme changes is easy since I already do it on the fly. Creating and maintaining logic for when the themes change is more work, and I'd like to rely on the OS where possible.

**Comment by heatl0rd** [comment]
How goes this feature to work with Android 10 global dark mode Allan?

Happy to donate to priorise this. Let me know.

**Comment by heatl0rd** [comment]
Hey everyone, I found a system wide app that provides this function on a per app basis. Requires Android 10 and root. Works very well!

https://labs.xda-developers.com/store/app/com.kieronquinn.app.darq

**Comment by AllanWang** [comment]
Does it work for frost? I haven't added settings for the day night toggle yet

**Comment by heatl0rd** [comment]
Yes it works on frost. The application has a inbuilt day night timer function based on time or location sunset details.

It only darkens and does not provide AMOLED.

I would lovs to see a native solution in frost to work with the android global dark system so we could customise the exact theme we want.

But I'll continue to be patient. Thanks Alan.
